flink-runtime/src/test/java/org/apache/flink/runtime/rpc/akka/MainThreadValidationTest.java

```diff
@@ -20,18 +20,19 @@
 
 import org.apache.flink.api.common.time.Time;
 import org.apache.flink.runtime.akka.AkkaUtils;
-
 import org.apache.flink.runtime.rpc.RpcEndpoint;
 import org.apache.flink.runtime.rpc.RpcGateway;
 import org.apache.flink.runtime.rpc.RpcService;
-
 import org.apache.flink.util.TestLogger;
+
+import org.junit.Ignore;
 import org.junit.Test;
 
 import java.util.concurrent.CompletableFuture;
 
 import static org.junit.Assert.assertTrue;
 
+@Ignore
```

Comment:
Why do we ignore this test?

Reply:
For both cases the test should be not ignored. Was a leftover.